logFilter.js

```diff
@@ -12,12 +12,22 @@ function filterLogsByEnv(logs, env) {
 
 function filterLogsOnDate(logs, dateFilter) {
   const filteredLogs = []
-  const parsedDateFilter = new Date(dateFilter)
-
-  logs.forEach((log) => {
-    const timestampDate = new Date(log.timestamp)
-
-    if (parsedDateFilter.toDateString() === timestampDate.toDateString()) {
+  let parsedDateFilter = new Date(dateFilter)
+  parsedDateFilter = new Date(
+    parsedDateFilter.getFullYear(),
+    parsedDateFilter.getUTCDate(),
+    parsedDateFilter.getUTCDay()
+  )
```

Comment:
When we create a date object with no time value it defaults to midnight. Date objects are always stored in UTC. So line 15 creates a date object set to midnight UTC on March 28, 2019. 

The new date you are creating is actually not going to be correct. Date() expects year, month, day. You are giving it 2019, 28, 4 since you are using getUTCDate and getUTCDay. That actually gives you 2021-05-04T04:00:00.000Z because the 28 for month makes everything wonky.